### ISSUE TITLE
PCAP writing I/O error

### ISSUE THREAD

**Issue by puniaze** [cuckoosandbox/cuckoo#1299]
Hi, I installed cuckoo on KVM to make it portable (Ubuntu 16.04 is running inside VBOX and cuckoo is installed to QEMU KVM). I followed guidlines and did what is needed. I wrote simple C application that pings HOST machine from Guest machine but cuckoo fails when it needs to write PCAP to storage folder and therefore I get empty PCAP output. (ERROR: Unable to open /home/cuckoo/cuckoo/storage/analyses/3/dump_sorted.pcap). My tcpdump is configured to run as non-root user and when I ping from Guest to Host machine it catches PING Echo and Replies perfectly.(tcpdump -i br0) . Here is cuckoo with debug enabled terminal output(some repeated lines omitted for simplicity):

Cuckoo with debug enabled terminal output
--------------------------------------
```
cuckoo@malwlab-VirtualBox:~/cuckoo$ python cuckoo.py -d

                     _ 
    ____ _   _  ____| |  _ ___   ___
   / ___) | | |/ ___) |_/ ) _ \ / _ \
  ( (___| |_| ( (___|  _ ( |_| | |_| |
   \____)____/ \____)_| \_)___/ \___/

 Cuckoo Sandbox 2.0-rc2
 www.cuckoosandbox.org
 Copyright (c) 2010-2015

 Checking for updates...
 Good! You have the latest version available.

2017-02-25 17:20:27,453 [root] DEBUG: Importing modules...
/usr/local/lib/python2.7/dist-packages/pymisp/api.py:18: UserWarning: You're using python 2, it is strongly recommended to use python >=3.3
  warnings.warn("You're using python 2, it is strongly recommended to use python >=3.3")
/usr/local/lib/python2.7/dist-packages/pymisp/mispevent.py:26: UserWarning: You're using python 2, it is strongly recommended to use python >=3.3
  warnings.warn("You're using python 2, it is strongly recommended to use python >=3.3")
/usr/local/lib/python2.7/dist-packages/pymisp/api.py:36: UserWarning: You're using python 2, it is strongly recommended to use python >=3.3
  warnings.warn("You're using python 2, it is strongly recommended to use python >=3.3")
2017-02-25 17:20:27,975 [root] DEBUG: Imported "signatures" modules:
2017-02-25 17:20:27,976 [root] DEBUG: 	 |-- CreatesExe
2017-02-25 17:20:27,976 [root] DEBUG: 	 `-- SystemMetrics
2017-02-25 17:20:27,976 [root] DEBUG: Imported "processing" modules:
2017-02-25 17:20:27,976 [root] DEBUG: 	 |-- AnalysisInfo
2017-02-25 17:20:27,977 [root] DEBUG: 	 |-- MetaInfo
2017-02-25 17:20:27,977 [root] DEBUG: 	 |-- ApkInfo
2017-02-25 17:20:27,977 [root] DEBUG: 	 |-- Baseline
2017-02-25 17:20:27,977 [root] DEBUG: 	 |-- BehaviorAnalysis
2017-02-25 17:20:27,978 [root] DEBUG: 	 |-- DroppedBuffer
2017-02-25 17:20:27,980 [root] DEBUG: 	 |-- Debug
2017-02-25 17:20:27,980 [root] DEBUG: 	 |-- Droidmon
2017-02-25 17:20:27,981 [root] DEBUG: 	 |-- Dropped
2017-02-25 17:20:27,981 [root] DEBUG: 	 |-- TLSMasterSecrets
2017-02-25 17:20:27,981 [root] DEBUG: 	 |-- GooglePlay
2017-02-25 17:20:27,981 [root] DEBUG: 	 |-- Irma
2017-02-25 17:20:27,982 [root] DEBUG: 	 |-- Memory
2017-02-25 17:20:27,982 [root] DEBUG: 	 |-- MISP
2017-02-25 17:20:27,982 [root] DEBUG: 	 |-- NetworkAnalysis
2017-02-25 17:20:27,982 [root] DEBUG: 	 |-- NetworkAnalysis
2017-02-25 17:20:27,983 [root] DEBUG: 	 |-- ProcessMemory
2017-02-25 17:20:27,983 [root] DEBUG: 	 |-- Procmon
2017-02-25 17:20:27,983 [root] DEBUG: 	 |-- Screenshots
2017-02-25 17:20:27,983 [root] DEBUG: 	 |-- Snort
2017-02-25 17:20:27,983 [root] DEBUG: 	 |-- Static
2017-02-25 17:20:27,983 [root] DEBUG: 	 |-- Strings
2017-02-25 17:20:27,984 [root] DEBUG: 	 |-- Suricata
2017-02-25 17:20:27,984 [root] DEBUG: 	 |-- TargetInfo
2017-02-25 17:20:27,984 [root] DEBUG: 	 `-- VirusTotal
2017-02-25 17:20:27,984 [root] DEBUG: Imported "auxiliary" modules:
2017-02-25 17:20:27,984 [root] DEBUG: 	 |-- MITM
2017-02-25 17:20:27,985 [root] DEBUG: 	 |-- Reboot
2017-02-25 17:20:27,985 [root] DEBUG: 	 |-- Services
2017-02-25 17:20:27,986 [root] DEBUG: 	 `-- Sniffer
2017-02-25 17:20:27,993 [root] DEBUG: Imported "reporting" modules:
2017-02-25 17:20:27,993 [root] DEBUG: 	 |-- ElasticSearch
2017-02-25 17:20:27,993 [root] DEBUG: 	 |-- JsonDump
2017-02-25 17:20:27,994 [root] DEBUG: 	 |-- Mattermost
2017-02-25 17:20:27,994 [root] DEBUG: 	 |-- Moloch
2017-02-25 17:20:27,994 [root] DEBUG: 	 |-- MongoDB
2017-02-25 17:20:27,994 [root] DEBUG: 	 |-- Notification
2017-02-25 17:20:28,000 [root] DEBUG: 	 `-- ReportHTML
2017-02-25 17:20:28,000 [root] DEBUG: Imported "machinery" modules:
2017-02-25 17:20:28,001 [root] DEBUG: 	 `-- KVM
2017-02-25 17:20:28,004 [root] DEBUG: Checking for locked tasks..
2017-02-25 17:20:28,031 [root] DEBUG: Checking for pending service tasks..
2017-02-25 17:20:28,061 [root] DEBUG: Initializing Yara...
2017-02-25 17:20:28,063 [root] DEBUG: 	 |-- index_binaries.yar
2017-02-25 17:20:28,065 [root] DEBUG: 	 `-- index_memory.yar
2017-02-25 17:20:28,079 [lib.cuckoo.core.resultserver] DEBUG: ResultServer running on 10.82.0.133:2042.
2017-02-25 17:20:28,081 [lib.cuckoo.core.scheduler] INFO: Using "kvm" as machine manager
2017-02-25 17:20:28,253 [lib.cuckoo.common.abstracts] DEBUG: Getting status for winxp-cucko0
2017-02-25 17:20:28,321 [lib.cuckoo.core.scheduler] INFO: Loaded 1 machine/s
2017-02-25 17:20:28,366 [lib.cuckoo.core.scheduler] INFO: Waiting for analysis tasks.
2017-02-25 17:20:42,958 [lib.cuckoo.core.scheduler] DEBUG: Processing task #3
2017-02-25 17:20:42,979 [lib.cuckoo.core.scheduler] INFO: Starting analysis of FILE "qul(1).exe" (task #3, options "route=none,procmemdump=yes")
2017-02-25 17:20:43,018 [lib.cuckoo.core.scheduler] INFO: File already exists at "/home/cuckoo/cuckoo/storage/binaries/f6bbd3438833b88dc16f43e5195b584afed847c8eb9ce267f4e4e937a8452f27"
2017-02-25 17:20:43,064 [lib.cuckoo.core.scheduler] INFO: Task #3: acquired machine cuckoo1 (label=winxp-cucko0)
2017-02-25 17:20:43,077 [modules.auxiliary.sniffer] INFO: Started sniffer with PID 30944 (interface=virbr0, host=10.82.0.46, pcap=/home/cuckoo/cuckoo/storage/analyses/3/dump.pcap)
2017-02-25 17:20:43,082 [lib.cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2017-02-25 17:20:43,185 [lib.cuckoo.common.abstracts] DEBUG: Starting machine winxp-cucko0
2017-02-25 17:20:43,185 [lib.cuckoo.common.abstracts] DEBUG: Getting status for winxp-cucko0
2017-02-25 17:20:43,235 [lib.cuckoo.common.abstracts] DEBUG: Using snapshot snap-agent-last-version for virtual machine winxp-cucko0
2017-02-25 17:20:46,340 [lib.cuckoo.common.abstracts] DEBUG: Getting status for winxp-cucko0
2017-02-25 17:20:46,407 [lib.cuckoo.core.guest] INFO: Starting analysis on guest (id=cuckoo1, ip=10.82.0.46)
2017-02-25 17:20:46,734 [lib.cuckoo.core.guest] DEBUG: cuckoo1: waiting for status 0x0001
2017-02-25 17:20:46,835 [lib.cuckoo.core.guest] DEBUG: cuckoo1: status ready
2017-02-25 17:20:46,904 [lib.cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=cuckoo1, ip=10.82.0.46, monitor=latest, size=1756407)
2017-02-25 17:20:52,168 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analyzer started with PID 1568
2017-02-25 17:20:52,218 [lib.cuckoo.core.guest] DEBUG: cuckoo1: waiting for completion
2017-02-25 17:20:53,241 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:20:54,300 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:20:55,352 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:20:56,423 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:20:57,381 [lib.cuckoo.core.resultserver] DEBUG: LogHandler for live analysis.log initialized.
..
..
..
2017-02-25 17:21:12,148 [lib.cuckoo.core.resultserver] DEBUG: New process (pid=720, ppid=1064, name=qul(1).exe)
2017-02-25 17:21:12,886 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:13,898 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:14,117 [lib.cuckoo.core.resultserver] DEBUG: New process (pid=1844, ppid=720, name=cmd.exe)
2017-02-25 17:21:14,912 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:15,693 [lib.cuckoo.core.resultserver] DEBUG: New process (pid=664, ppid=1844, name=ping.exe)
2017-02-25 17:21:15,965 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:16,979 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
..
..
..
2017-02-25 17:21:32,220 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:32,689 [lib.cuckoo.core.resultserver] DEBUG: File upload request for memory/664-1.dmp
2017-02-25 17:21:33,509 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:34,704 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:35,876 [lib.cuckoo.core.resultserver] WARNING: Uploaded file length larger than upload_max_size, stopping upload.
2017-02-25 17:21:35,876 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 10485775
2017-02-25 17:21:35,980 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:37,349 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:38,378 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:39,393 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:40,417 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:41,430 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:41,881 [lib.cuckoo.core.resultserver] DEBUG: File upload request for memory/1844-1.dmp
2017-02-25 17:21:42,608 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:43,631 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:44,170 [lib.cuckoo.core.resultserver] WARNING: Uploaded file length larger than upload_max_size, stopping upload.
2017-02-25 17:21:44,171 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 10485775
2017-02-25 17:21:44,648 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:45,672 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:46,689 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:21:47,022 [lib.cuckoo.core.resultserver] DEBUG: New process (pid=1028, ppid=720, name=cmd.exe)
2017-02-25 17:21:47,701 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
..
..
..
2017-02-25 17:23:11,510 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:23:12,522 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:23:13,174 [lib.cuckoo.core.resultserver] DEBUG: File upload request for memory/720-1.dmp
2017-02-25 17:23:14,876 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 7113176
2017-02-25 17:23:15,688 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:23:16,737 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:23:20,343 [lib.cuckoo.core.resultserver] DEBUG: File upload request for memory/1028-1.dmp
2017-02-25 17:23:21,092 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:23:22,683 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:23:23,389 [lib.cuckoo.core.resultserver] WARNING: Uploaded file length larger than upload_max_size, stopping upload.
2017-02-25 17:23:23,389 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 10485775
2017-02-25 17:23:23,696 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2017-02-25 17:23:24,708 [lib.cuckoo.core.guest] INFO: cuckoo1: analysis completed successfully
2017-02-25 17:23:24,719 [lib.cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Sniffer
2017-02-25 17:23:24,721 [lib.cuckoo.common.abstracts] DEBUG: Stopping machine winxp-cucko0
2017-02-25 17:23:24,722 [lib.cuckoo.common.abstracts] DEBUG: Getting status for winxp-cucko0
2017-02-25 17:23:25,645 [lib.cuckoo.common.abstracts] DEBUG: Getting status for winxp-cucko0
2017-02-25 17:23:25,723 [lib.cuckoo.core.scheduler] DEBUG: Released database task #3
2017-02-25 17:23:25,781 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:26,556 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:26,595 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:26,595 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:26,603 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Debug" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:26,603 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:26,604 [lib.cuckoo.common.objects] WARNING: Unable to import yara (please compile from sources)
2017-02-25 17:23:26,891 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:26,891 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:27,305 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Static" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:27,313 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Strings" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:27,316 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:27,318 [lib.cuckoo.core.plugins] DEBUG: Processing module network-backup not found in configuration file
2017-02-25 17:23:27,319 [modules.processing.network] DEBUG: Whitelisting Disabled.
2017-02-25 17:23:27,319 [modules.processing.network] ERROR: Unable to open /home/cuckoo/cuckoo/storage/analyses/3/dump_sorted.pcap
2017-02-25 17:23:27,346 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:29,322 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "VirusTotal" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:29,324 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" on analysis at "/home/cuckoo/cuckoo/storage/analyses/3"
2017-02-25 17:23:29,325 [lib.cuckoo.core.plugins] DEBUG: Running 0 signatures
2017-02-25 17:23:29,726 [lib.cuckoo.core.plugins] DEBUG: Executed reporting module "JsonDump"
2017-02-25 17:23:29,882 [lib.cuckoo.core.plugins] DEBUG: Executed reporting module "MongoDB"
2017-02-25 17:23:29,883 [lib.cuckoo.core.scheduler] INFO: Task #3: reports generation completed (path=/home/cuckoo/cuckoo/storage/analyses/3)
2017-02-25 17:23:29,933 [lib.cuckoo.core.scheduler] INFO: Task #3: analysis procedure completed
```

This is my cuckoo.conf
------------------------
```
[cuckoo]
# Enable or disable startup version check. When enabled, Cuckoo will connect
# to a remote location to verify whether the running version is the latest
# one available.
version_check = on

# If turned on, Cuckoo will delete the original file after its analysis
# has been completed.
delete_original = off

# If turned on, Cuckoo will delete the copy of the original file in the
# local binaries repository after the analysis has finished. (On *nix this
# will also invalidate the file called "binary" in each analysis directory,
# as this is a symlink.)
delete_bin_copy = off

# Specify the name of the machinery module to use, this module will
# define the interaction between Cuckoo and your virtualization software
# of choice.
machinery = kvm

# Enable creation of memory dump of the analysis machine before shutting
# down. Even if turned off, this functionality can also be enabled at
# submission. Currently available for: VirtualBox and libvirt modules (KVM).
memory_dump = off

# When the timeout of an analysis is hit, the VM is just killed by default.
# For some long-running setups it might be interesting to terminate the
# monitored processes before killing the VM so that connections are closed.
terminate_processes = off

# Enable automatically re-schedule of "broken" tasks each startup.
# Each task found in status "processing" is re-queued for analysis.
reschedule = off

# Enable processing of results within the main cuckoo process.
# This is the default behavior but can be switched off for setups that
# require high stability and process the results in a separate task.
process_results = on

# Limit the amount of analysis jobs a Cuckoo process goes through.
# This can be used together with a watchdog to mitigate risk of memory leaks.
max_analysis_count = 0

# Limit the number of concurrently executing analysis machines.
# This may be useful on systems with limited resources.
# Set to 0 to disable any limits.
max_machines_count = 0

# Limit the amount of VMs that are allowed to start in parallel. Generally
# speaking starting the VMs is one of the more CPU intensive parts of the
# actual analysis. This option tries to avoid maxing out the CPU completely.
max_vmstartup_count = 10

# Minimum amount of free space (in MB) available before starting a new task. 
# This tries to avoid failing an analysis because the reports can't be written 
# due out-of-diskspace errors. Setting this value to 0 disables the check.
# (Note: this feature is currently not supported under Windows.)
freespace = 64

# Temporary directory containing the files uploaded through Cuckoo interfaces
# (api.py and Django web interface).
tmppath = /tmp

# Path to the unix socket for running root commands.
rooter = /tmp/cuckoo-rooter

[routing]
# Default network routing mode; "none", "internet", or "vpn_name".
# In none mode we don't do any special routing - the VM doesn't have any
# network access (this has been the default actually for quite a while).
# In internet mode by default all the VMs will be routed through the network
# interface configured below (the "dirty line").
# And in VPN mode by default the VMs will be routed through the VPN identified
# by the given name of the VPN (as per vpn.conf).
# Note that just like enabling VPN configuration setting this option to
# anything other than "none" requires one to run utils/rooter.py as root next
# to the Cuckoo instance (as it's required for setting up the routing).
route = none

# Network interface that allows a VM to connect to the entire internet, the
# "dirty line" so to say. Note that, just like with the VPNs, this will allow
# malicious traffic through your network. So think twice before enabling it.
# (For example, to route all VMs through eth0 by default: "internet = eth0").
internet = none

# Routing table name/id for "dirty line" interface. If "dirty line" is
# also default gateway in the system you can leave "main" value. Otherwise add
# new routing table by adding "<id> <name>" line to /etc/iproute2/rt_tables
# (e.g., "200 eth0"). ID and name must be unique across the system (refer to
# /etc/iproute2/rt_tables for existing names and IDs).
rt_table = main

# To route traffic through multiple network interfaces Cuckoo uses 
# Policy Routing with separate routing table for each output interface
# (VPN or "dirty line"). If this option is enabled Cuckoo on start will try 
# to automatically initialise routing tables by copying routing entries from 
# main routing table to the new routing tables. Depending on your network/vpn 
# configuration this might not be sufficient. In such case you would need to 
# initialise routing tables manually. Note that enabling this option won't
# affect main routing table.
auto_rt = yes

[resultserver]
# The Result Server is used to receive in real time the behavioral logs
# produced by the analyzer.
# Specify the IP address of the host. The analysis machines should be able
# to contact the host through such address, so make sure it's valid.
# NOTE: if you set resultserver IP to 0.0.0.0 you have to set the option
# `resultserver_ip` for all your virtual machines in machinery configuration.
ip = 10.82.0.133

# Specify a port number to bind the result server on.
port = 2042

# Force the port chosen above, don't try another one (we can select another
# port dynamically if we can not bind this one, but that is not an option
# in some setups)
force_port = no

# Maximum size of uploaded files from VM (screenshots, dropped files, log)
# The value is expressed in bytes, by default 10Mb.
upload_max_size = 10485760

[processing]
# Set the maximum size of analyses generated files to process. This is used
# to avoid the processing of big files which may take a lot of processing
# time. The value is expressed in bytes, by default 100Mb.
analysis_size_limit = 104857600

# Enable or disable DNS lookups.
resolve_dns = on

# Enable PCAP sorting, needed for the connection content view in the web interface.
sort_pcap = on

[database]
# Specify the database connection string.
# NOTE: If you are using a custom database (different from sqlite), you have to
# use utf-8 encoding when issuing the SQL database creation statement.
# Examples, see documentation for more:
# sqlite:///foo.db
# postgresql://foo:bar@localhost:5432/mydatabase
# mysql://foo:bar@localhost/mydatabase
# If empty, default is a SQLite in db/cuckoo.db.
connection =

# Database connection timeout in seconds.
# If empty, default is set to 60 seconds.
timeout =

[timeouts]
# Set the default analysis timeout expressed in seconds. This value will be
# used to define after how many seconds the analysis will terminate unless
# otherwise specified at submission.
default = 120

# Set the critical timeout expressed in (relative!) seconds. It will be added
# to the default timeout above and after this timeout is hit
# Cuckoo will consider the analysis failed and it will shutdown the machine
# no matter what. When this happens the analysis results will most likely
# be lost.
critical = 60

# Maximum time to wait for virtual machine status change. For example when
# shutting down a vm. Default is 60 seconds.
vm_state = 60
```

----------------

Ifconfig output
------------------

```
cuckoo@malwlab-VirtualBox:~/cuckoo/conf$ ifconfig 
br0       Link encap:Ethernet  HWaddr 08:00:27:66:a6:0c  
          inet addr:10.82.0.133  Bcast:10.82.0.255  Mask:255.255.255.0
          inet6 addr: fe80::a00:27ff:fe66:a60c/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:330167 errors:0 dropped:9902 overruns:0 frame:0
          TX packets:140313 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:389715093 (389.7 MB)  TX bytes:22057027 (22.0 MB)

enp0s3    Link encap:Ethernet  HWaddr 08:00:27:66:a6:0c  
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:297902 errors:0 dropped:684 overruns:0 frame:0
          TX packets:72401 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:226993257 (226.9 MB)  TX bytes:5753215 (5.7 MB)

lo        Link encap:Local Loopback  
          inet addr:127.0.0.1  Mask:255.0.0.0
          inet6 addr: ::1/128 Scope:Host
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:103901 errors:0 dropped:0 overruns:0 frame:0
          TX packets:103901 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1 
          RX bytes:109628633 (109.6 MB)  TX bytes:109628633 (109.6 MB)

virbr0    Link encap:Ethernet  HWaddr 00:00:00:00:00:00  
          inet addr:192.168.122.1  Bcast:192.168.122.255  Mask:255.255.255.0
          UP BROADCAST MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
```

**Comment by doomedraven** [comment]
you sais `tcpdump -i br0` but in cuckoo log i see `virbr0` and virbr uses .122.0/24 netowork

**Comment by puniaze** [comment]
OMG yes you are right :) I changed as you said and now it works perfectly :) Thank u so much!

**Comment by doomedraven** [comment]
you are welcome :)
